x/programs/cmd/simulator/cmd/plan.go

```diff
@@ -169,7 +169,6 @@ func (c *runCmd) RunStep(ctx context.Context, db *state.SimpleMutable) (*Respons
 	resp := newResponse(index)
 	err = runStepFunc(ctx, c.log, db, step.Endpoint, step.MaxUnits, step.Method, params, resp)
 	if err != nil {
-		c.log.Error(fmt.Sprintf("simulation step err: %s", err))
```

Comment:
This is annoying, you get the same message on the Rust side anyway

Reply:
💯